### PR TITLE
Update typescript to 2.0

### PR DESCRIPTION
**Pull request by alexeagle**

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -130,6 +132,7 @@ function createSourceReplacingCompilerHost(
     fileExists: delegate.fileExists,
     readFile: delegate.readFile,
     directoryExists: delegate.directoryExists,
+    getDirectories: delegate.getDirectories,
```
Comment (alexeagle) [inline src/main.ts]: I checked that this works in current google3 TS version.

**Comment by alexeagle** [comment]
oh, now I understand the discussion about comment trivia at MS... :)

**Comment by evmar** [comment]
I can rebaseline at work tomorrow.

**Comment by evmar** [comment]
I fixed up this CL and landed a working version.

**Comment by alexeagle** [comment]
Super 🍊

**Pull request closed without merging**